modules/claims_api/app/views/claims_api/unsuccessful_report_mailer/_poa_status_table.html.erb

```diff
@@ -1,16 +1,22 @@
 <table>
   <thead>
     <tr>
+        <th>consumer</th>
         <% ClaimsApi::PowerOfAttorney::ALL_STATUSES.each do |status| %>
         <th><%= status %></th>
         <% end %>
+        <th>totals</th>
     </tr>
   </thead>
   <tbody>
+          <% poa_consumers.each do |consumer| %>
```

Comment:
Pretty sure this has to be referenced as `@poa_consumers`.

Reply:
Err, maybe not?  🤔 
The `_claims_status_table` partial refers to them without the `@` and those work, hahaaha.
I would've sworn you had to use the `@` but I guess not.  😂 🤷